docs/mp/openapi/openapi.adoc

```diff
@@ -85,10 +85,10 @@ include::{incdir}/openapi.adoc[tag=usage-access-endpoint]
 include::{incdir}/openapi.adoc[tag=api]
 
 == Configuration
-
-Helidon OpenAPI configuration supports the following settings:
-
-include::{rootdir}/config/io_helidon_microprofile_openapi_MPOpenAPISupport.adoc[leveloffset=+1,tag=config]
+//
```

Comment:
These lines or ones like them will need to come back once the doc is brought up-to-date again. For me it's more convenient to have the to-be-revised lines there in commented form rather than see an empty "Configuration" section and bring up the history to see what was there before.

Reply:
ok